src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,40 +1,78 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { TODO_ACTIONS, TODO_SELECTORS } from '../../features/currentTodo';
 import { Loader } from '../Loader';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { LOADING_ACTIONS, LOADING_SELECTORS } from '../../features/loading';
 
 export const TodoModal: React.FC = () => {
-  return (
-    <div className="modal is-active" data-cy="modal">
-      <div className="modal-background" />
+  const selectedTodo = useSelector(TODO_SELECTORS.getCurrentTodo);
 
-      <Loader />
+  const dispatch = useDispatch();
+  const isLoading = useSelector(LOADING_SELECTORS.getTodoLoadingStatus);
 
-      <div className="modal-card">
-        <header className="modal-card-head">
-          <div
-            className="modal-card-title has-text-weight-medium"
-            data-cy="modal-header"
-          >
-            Todo #3
-          </div>
+  const setIsLoading = (status: boolean) => {
+    dispatch(LOADING_ACTIONS.setTodoLoading(status));
+  };
+
+  const handleDeselectTodo = () => dispatch(TODO_ACTIONS.removeTodo());
 
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <button type="button" className="delete" data-cy="modal-close" />
-        </header>
+  const [foundUser, setFoundUser] = useState<User | null>(null);
```

Comment:
```suggestion
  const [user, setUser] = useState<User | null>(null);
```
Don't use condition in variable's name. You can also name it author/todoUser etc.